src/current/cockroachcloud/upgrade-to-v23.2.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Upgrade to CockroachDB v23.2 Pre-Production Preview
+summary: Learn how to upgrade your CockroachDB cluster to v23.2 Pre-Production Preview
+toc: true
+docs_area: manage
+page_version: v23.2
+prev_version: v23.1
+pre_production_preview: True
+pre_production_preview_version: v23.2.0-beta.3
+---
+
+{% if page.pre_production_preview == true %}
+[CockroachDB {{ page.pre_production_preview_version }}](https://www.cockroachlabs.com/docs/releases/{{ page.page_version }}#{{ page.pre_production_preview_version | replace: ".","-"}})  is available to CockroachDB {{ site.data.products.dedicated }} clusters for testing and experimentation. An [Org Administrator]({% link cockroachcloud/authorization.md %}#org-administrator-legacy) can upgrade your CockroachDB {{ site.data.products.dedicated }} cluster from the CockroachDB {{ site.data.products.cloud }} Console. This page guides you through the process of upgrading. To enroll your organization, [contact Support](https://support.cockroachlabs.com/hc/).
+
```

Comment:
How about a warning here to the effect that this (pre-prod preview and the testing version it enables) is not recommended or supported for upgrading a Production cluster, and is for testing purposes only?

Maybe we can reuse some of the existing language we have elsewhere?

Reply:
Now really done